recipes/daltools/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "daltools" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f3f21cfae3af36b4a941573470dd70a8910307204bbc0a3568d7db7c8ed7f8f9
+
+build:
+  number: 1
```

This should start from 0
```suggestion
  number: 0
```